odoo/tools/config.py

```diff
@@ -200,8 +200,10 @@ def _build_cli(self):
                          help="install one or more modules (comma-separated list, use \"all\" for all modules), requires -d")
         group.add_option("-u", "--update", dest="update", type='comma',  metavar="MODULE,...", my_default=[], file_loadable=False,
                          help="update one or more modules (comma-separated list, use \"all\" for all modules). Requires -d.")
-        group.add_option("--without-demo", dest="without_demo", my_default=False, type='without_demo', metavar='BOOL', nargs='?', const=True,
-                         help="use with -i/--init, skip installing fake demonstration data (e.g. Mitchel Admin/Azure Interior)")
+        group.add_option("--with-demo", dest="with_demo", action="store_true", my_default=False,
+                         help="Enable loading demo data for modules to be installed. Requires -d or -i. Default is %default")
+        group.add_option("--without-demo", dest="with_demo", my_default=False, type='without_demo', metavar='BOOL', nargs='?', const=False,
```

Comment:
You may need to keep `dest="without_demo"` as backwards compatibility.
Then compute "with_demo" in the runtime values as (with_demo or not without_demo).
@Julien00859 what do you think?

Reply:
we definitely need something to support old configuration files